taxonomies/ingredients.txt

```diff
@@ -27,6 +27,18 @@ synonyms:es:jarabe, sirope
 
 synonyms:fr:semi-complet, demi-complet
 
+synonyms:es:semi-completo, medio completo
+
+synonyms:de:halb-vollständig, halb-komplett
+
+synonyms:it:semi-completo, mezzo completo
+
+synonyms:pt:semi-completo, meio completo
+
+synonyms:nl:halfvol, half-compleet
+
+synonyms:ru:полу-полный, полу-комплексный
+
```

Comment:
Are you sure that it is a good idea?

In French, it is common to have both forms, they are synonyms.

For each other languages, none of both forms are found in the taxonomy...